index.html

```diff
@@ -8,10 +8,11 @@
     <script src="https://unpkg.com/kaboom/dist/kaboom.js"></script>
   </head>
   <body>
+    <div id="container">
+      <canvas id="game"></canvas>
```

Comment:
### Changing the <canvas> property name
The main thing here was that the canvas property should be "id" and not "name".
### Changing the container lexical placement
I'm not entirely sure, but I think having the container and canvas defined before running the script also changed things; I think kaboom was making its own canvas originally, which is why the kaboom canvas was always below the canvas we created.

Reply:
Interesting, i didn't have the issue with 2 canvases, you may have had that because of how we initialized kaboom but its fixed now.